common/browser_context.go

```diff
@@ -443,21 +443,16 @@ func (b *BrowserContext) getSession(id target.SessionID) *Session {
 
 // AddCookies adds cookies into this browser context.
 // All pages within this context will have these cookies installed.
-func (b *BrowserContext) AddCookies(cookies goja.Value) error {
+func (b *BrowserContext) AddCookies(cookies []*api.Cookie) error {
```

Comment:
Does goja automatically try to convert from it's internal type into `[]*api.Cookie`? I don't see a change in the mapping layer to handle that.

Reply:
Yes, it does. We don't need to change the mapping layer 🥳